.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,55 @@
+name: golangci-lint
+on:
+  push:
+    paths:
+      - 'gov2/**'
+    branches:
+      - main
+  pull_request:
+    paths:
+      - 'gov2/**'
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+      - uses: actions/checkout@v3
+      - name: install golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
```

Comment:
Optional: Version of golangci-lint to use. For example, v1.2 or v1.2.3, or `latest` to use the latest version.

Reply:
done